### ISSUE TITLE
CC required although there are enough APPC and ETH available for the transaction

### ISSUE THREAD

**Issue by nenovmy** [Aptoide/bds-unity-plugin#4]
In my game Follow the Line 2D Deluxe some prices in the store are shown in APPC and EUR (when you click on the item to buy it). When I try to buy this item a CC input dialog is opened although there are enough APPC and ETH in my wallet!
Other Items have prices only in APPC and can be purchases without problem.

I thin the first items are items that I tried to purchase long ago when I didn't have enough ETH and now for some reason the plugin still wants a CC.

**Comment by nunomonteiro** [comment]
Hi nenovmy,

Sorry for the inconvenience.
Do you have the latest version of the wallet installed?
We're analysing the issue internally and will get back to you ASAP.

Cheers,
Nuno

**Comment by nenovmy** [comment]
Sorry for the missing information! Here is everything:

Unity version: 2018.2.13f1
BDS unity plugin: 0.3
Wallet version: 1.1.1.4

Testing on Samsung galaxy s7 edge with Android 8.0.0

**Comment by nenovmy** [comment]
I upgraded to the latest BDS plugin 1.0.1. I created a new item and was able to make a purchase, but after the success the app crashed and now crashes on every start like the issue #5:

10-23 15:29:05.816: E/ViewRootImpl(32086): sendUserActionEvent() returned.
10-23 15:29:05.817: E/ActivityThread(32086): Activity com.test.notifdetect.OverriddenUpa has leaked ServiceConnection null that was originally bound here
10-23 15:29:05.817: E/ActivityThread(32086): android.app.ServiceConnectionLeaked: Activity com.test.notifdetect.OverriddenUpa has leaked ServiceConnection null that was originally bound here
10-23 15:29:05.817: E/ActivityThread(32086): 	at android.app.LoadedApk$ServiceDispatcher.<init>(LoadedApk.java:1538)
10-23 15:29:05.817: E/ActivityThread(32086): 	at android.app.LoadedApk.getServiceDispatcher(LoadedApk.java:1430)
10-23 15:29:05.817: E/ActivityThread(32086): 	at android.app.ContextImpl.bindServiceCommon(ContextImpl.java:1622)
10-23 15:29:05.817: E/ActivityThread(32086): 	at android.app.ContextImpl.bindService(ContextImpl.java:1574)
10-23 15:29:05.817: E/ActivityThread(32086): 	at android.content.ContextWrapper.bindService(ContextWrapper.java:697)
10-23 15:29:05.817: E/ActivityThread(32086): 	at com.unity3d.player.UnityPlayer.nativeRender(Native Method)
10-23 15:29:05.817: E/ActivityThread(32086): 	at com.unity3d.player.UnityPlayer.c(Unknown Source:0)
10-23 15:29:05.817: E/ActivityThread(32086): 	at com.unity3d.player.UnityPlayer$e$2.queueIdle(Unknown Source:72)
10-23 15:29:05.817: E/ActivityThread(32086): 	at android.os.MessageQueue.next(MessageQueue.java:394)
10-23 15:29:05.817: E/ActivityThread(32086): 	at android.os.Looper.loop(Looper.java:142)
10-23 15:29:05.817: E/ActivityThread(32086): 	at com.unity3d.player.UnityPlayer$e.run(Unknown Source:32)

**Comment by nunomonteiro** [comment]
Hello,
Sorry for the late reply. After internal talks with the team, I figured this issue is related to the wallet and not the plugin by itself.
Could you please post it here: 
https://github.com/Aptoide/appcoins-wallet-android/issues/
Thanks!

**Comment by nenovmy** [comment]
Done!

**Comment by nenovmy** [comment]
Hi Nuno,

how are you?
I just saw that there is an issue with the price calculation in the BDS store.

What I did:
* First I set all the prices in APPC (before there was an option to retrieve the price in the API)
* Later I created a new Item and set the price in USD, I also switched all other prices to USD, so the APPC price is calculated automatically based on the current APPC value

Expected:
All prices are shown in APPC calculated from the USD value and current APPC price

Actual:
* The price for the new item I created in USD is calculated as expected
* The prices for the old items I switched from APPC to USD are not calculate at all, but instead the USD price is taken without conversion as price in APPC (e.g. $4.55 is shown as 4.55 APPC)

Thanks,
Matey
